cluster/manifests/skipper/hpa.yaml

```diff
@@ -53,6 +53,19 @@ spec:
       target:
         averageValue: {{ .ConfigItems.skipper_scaling_schedule_hyped_articles_target }}
         type: AverageValue
+{{ end }}
+{{ if and (ne .ConfigItems.skipper_scaling_schedule_custom_name "") (ne .ConfigItems.skipper_scaling_schedule_custom_target "") }}
```

Comment:
Maybe we would want to have a list of schedules such that we could subscribe to more schedules if needed? E.g. some clusters may want to subscribe to the lounge AND the hyped article schedules?

Reply:
>  E.g. some clusters may want to subscribe to the lounge AND the hyped article schedules?

This particular example is possible since we keep hyped schedule. But if you mean a generic scheme then maybe we should go for something extensible like
`skipper_scaling_schedule_1_name and skipper_scaling_schedule_1_target`, `skipper_scaling_schedule_2_name and skipper_scaling_schedule_2_target`.
WDYT?

Reply:
Yeah, either that or encoded in one string list: `<name>=<target>,...` and then decode the string and loop through it. Maybe it's too complicated to configure though? :) We do it for some other use cases in this way.

Reply:
I've updated as suggested